### DISCUSSION TITLE
SMS

### DISCUSSION THREAD

**Discussion by Ruslanm001** [community/community#112600] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am writing to report an issue regarding the SMS verification process for my GitHub account.

Unfortunately, I recently changed my phone number, and as a result, I am unable to complete the SMS verification process to access my account. This has left me unable to log in and access the resources and projects associated with my GitHub account.

I would greatly appreciate your assistance in resolving this matter promptly. Please let me know what steps I can take to regain access to my account, considering the change in my phone number.

with another email ([ruslanm0502@gmail.com](mailto:ruslanm0502@gmail.com))

**Comment by DNin01** [comment, marked as answer]
Do you still have your account's recovery codes? You may have saved them when you set up 2FA. If so, you can choose "Use a recovery code or request a reset" and enter one in, as a text message alternative. Once you're logged in, update the phone number. You can learn more about recovering an account with 2FA enabled in the [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials).

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Ruslanm001! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
